src/extensions/default/SVGCodeHints/SVGUtils.js

```diff
@@ -0,0 +1,339 @@
+/*
+ * Copyright (c) 2014 Amin Ullah Khan. All rights reserved.
```

Comment:
I don't know either, but other files like [ThemeManager](https://github.com/adobe/brackets/blob/master/src/view/ThemeManager.js) don't have `Adobe Systems` in it as well.